src/upstream/types/platform.rs

```diff
@@ -176,6 +176,12 @@ pub enum Platform {
     #[graphql(name = "tron")]
     Tron,
 
+    /// Tron
```

Comment:
TON or Tron?

Reply:
Also leave full product name here since it will be displayed in GraphQL API doc

Reply:
Both `TON Network` and `TRON Network` are exist.
I have put the product description and official website of each platform to the platform enumeration annotation 😊